cms/djangoapps/contentstore/models.py

```diff
@@ -0,0 +1,32 @@
+"""
+Models for contentstore
+"""
+
+from django.db.models.fields import TextField
+
+from config_models.models import ConfigurationModel
+
+
+class VideoEncodingDownloadConfig(ConfigurationModel):
```

Comment:
So should this be in a different directory than contentstore (peer with contentstore)?